src/vcpkg/base/system.cpp

```diff
@@ -8,6 +8,13 @@
 
 using namespace vcpkg::System;
 
+#if defined(__APPLE__)
+extern "C"
+{
+    int sysctlbyname(const char*, void*, size_t*, void*, size_t);
```

Comment:
It seems like it would be better to include the right headers ( https://developer.apple.com/library/archive/documentation/System/Conceptual/ManPages_iPhoneOS/man3/sysctlbyname.3.html ) rather than declaring this?

Reply:
I couldn't find the header, tbh...